types/wlr_xdg_shell_v6.c

```diff
@@ -948,6 +964,7 @@ static void wlr_xdg_toplevel_v6_send_configure(
 	zxdg_toplevel_v6_send_configure(surface->toplevel_state->resource, width,
 		height, &states);
 
+out:
 	wl_array_release(&states);
```

Comment:
We should probably accompany these cases by posting OOM to the client

Reply:
I've seen `wl_resource_post_no_memory` around, is that what you are referring to? What does that actually does?

Reply:
Yes, that's what I'm talking about. It tells the client that we were unable to fulfill its request due for OOMy reasons.

Reply:
Ok, I believe something like that might work. Not sure how to test in practice as this can't actually happen on linux/glibc... Hope for the best?